packages/backend/server/src/fundamentals/utils/request.ts

```diff
@@ -66,3 +66,26 @@ export function getRequestFromHost(host: ArgumentsHost) {
 export function getRequestResponseFromContext(ctx: ExecutionContext) {
   return getRequestResponseFromHost(ctx);
 }
+
+export function parseCookies(req: Request) {
```

Comment:
need some function comments to explain the behavior that will overwrite req.cookies